README.md

```diff
@@ -38,42 +38,42 @@ var coffeeFile = new File({
 
 #### options.cwd
 
-Type: `String`  
```

Comment:
Markdown treats double trailing whitespace as a newline. It's stupid, but @stevemao is right.